bundle/internal/schema/annotations.yml

```diff
@@ -0,0 +1,325 @@
+github.com/databricks/cli/bundle/config.Artifact:
+  description: Defines the attributes to build an artifact.
+  title: Artifact
+github.com/databricks/cli/bundle/config.Artifact.build:
```

Comment:
What do you think about grouping the struct fields together? Something like: 

```
github.com/databricks/cli/bundle/config.Artifact:
  executable:
    description: PLACEHOLDER
  files:
    description: PLACEHOLDER
  path:
    description: PLACEHOLDER
  type:
    description: PLACEHOLDER
```

It'll make this file easier to read and maintain. We don't need any docs for the top level `Root{}` struct so we are not missing out on completeness.